### PR TITLE
[AAE-10308] Karma process-service-cloud 404 messages

### PR DESCRIPTION
**Pull request by marcol13**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-10308

**What is the new behaviour?**
Warning messages are gone

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/karma.conf.js [hunk 1]
```diff
@@ -31,13 +31,20 @@ module.exports = function (config) {
         proxies: {
             '/assets/': '/base/lib/process-services-cloud/src/lib/assets/',
             '/resources/i18n/en.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
+            '/resources/i18n/en-US.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
             '/resources/i18n/fr.json': '/base/lib/process-services-cloud/src/lib/mock/fr.json',
             '/base/assets/': '/base/lib/process-services/assets/',
+            '/base/lib/core/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
+            '/base/lib/process-services-cloud/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
```
Comment (mauriziovitale) [inline lib/process-services-cloud/karma.conf.js:38]: why are we passing the core as process services cloud ?

**Review by mauriziovitale — CHANGES_REQUESTED** [review]
Please check it better

File: lib/process-services-cloud/karma.conf.js [hunk 2]
```diff
@@ -31,13 +31,20 @@ module.exports = function (config) {
         proxies: {
             '/assets/': '/base/lib/process-services-cloud/src/lib/assets/',
             '/resources/i18n/en.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
+            '/resources/i18n/en-US.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
             '/resources/i18n/fr.json': '/base/lib/process-services-cloud/src/lib/mock/fr.json',
             '/base/assets/': '/base/lib/process-services/assets/',
+            '/base/lib/core/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
```
[the remark this answers is not in the document]
Comment (alep85) [reply lib/process-services-cloud/karma.conf.js:37]: Without that rule the warnings are showed during tests execution
image

[the remark this answers is not in the document]
Comment (marcol13) [reply lib/process-services-cloud/karma.conf.js:37]: Yes, I have the same issue without that line - I am trying to workaround it, but for now every of my attempts failed.

File: lib/process-services-cloud/karma.conf.js (hunk 1 above)
Comment (marcol13) [reply lib/process-services-cloud/karma.conf.js:38]: Line 38 requires content which is available only in core library - without it I have these errors:
```
[33m27 09 2022 10:06:55.891:WARN [web-server]: [39m404: /_karma_webpack_/base/lib/process-services-cloud/src/lib/assets/images/ft_ic_raster_image.svg
```

File: lib/process-services-cloud/karma.conf.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrea-ligios) [reply lib/process-services-cloud/karma.conf.js:37]: Please @marcol13 (and @alep85 / @mauriziovitale), take a look at this and see if it works: 
https://dev.to/jwp/angular-karma-proxy-configuration-1kbb

[the remark this answers is not in the document]
Comment (marcol13) [reply lib/process-services-cloud/karma.conf.js]: Resolved in c0efcb122c91f7f93065de07f75f700e43d8b579

File: lib/process-services-cloud/karma.conf.js [hunk 3]
```diff
@@ -31,13 +31,20 @@ module.exports = function (config) {
...
+            '/base/lib/process-services-cloud/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
             '/assets/adf-core/i18n/en.json': '/base/lib/core/src/lib/i18n/en.json',
             '/assets/adf-core/i18n/en-GB.json': '/base/lib/core/src/lib/i18n/en.json',
+            '/assets/adf-core/i18n/en-US.json': '/base/lib/core/src/lib/i18n/en.json',
+            '/assets/adf-core/i18n/fr.json': '/base/lib/core/src/lib/i18n/fr.json',
+            '/base/lib/core/src/lib/i18n/fr.json': '/base/lib/core/src/lib/i18n/fr.json',
             '/assets/adf-process-services-cloud/i18n/en.json': '/base/lib/process-services-cloud/lib/i18n/en.json',
             '/assets/adf-process-services-cloud/i18n/en-GB.json': '/base/lib/process-services-cloud/lib/i18n/en.json',
             '/app.config.json': '/base/lib/config/app.config.json',
+            '/ecm/alfresco/api/-default-/public/alfresco/versions/1/nodes/-my-': ''
```
[the remark this answers is not in the document]
Comment (marcol13) [reply lib/process-services-cloud/karma.conf.js]: @mauriziovitale Is it really necessary? Maybe changing value of it to: `http://localhost:9876/ecm/alfresco/api/-default-/public/alfresco/versions/1/nodes/-my-`  will be enough? I don't get any warnings/errors after doing that.

**Comment by andrea-ligios** [comment]
This is 80 commits behind and the effort for resolving the conflicts now is much greater than the benefits of the code itself

**Pull request closed without merging**